assets/pages/contributions-landing/contributionsLandingUS.jsx

```diff
@@ -42,7 +49,11 @@ const content = (
       <SimpleHeader />
       <section className="contributions-bundle">
         <div className="contributions-bundle__content gu-content-margin">
-          <ContributionsIntroduction />
+          {store.getState().contribution.context
```

Comment:
Discussed in real life, but would the view change if the state was changed after the initial render?

Reply:
I've put this into a connected component now.